index.js

```diff
@@ -48,8 +49,8 @@ var complete = function(phase) {
     return Promise.resolve();
   }
 
-  return phase
-    .reportTest()
+  return Promise
```

Comment:
I'm confused as to why a promise is used here... just my knowledge of promises...